release-notes/major11/11.2.0/valtimo-backend-libraries.md

```diff
@@ -93,6 +101,11 @@ The following bugs were fixed:
   Date search fields now take into account the timezone offset sent along with requests under the `X-Timezone-Offset`
   request header. If this header is not present, UTC+0 is used.
 
+* **Error in some plugin action properties**
+
+  A bug has been fixed where the plugin action property would throw an error when the plugin action property was set to
```

Comment:
Please only describe the bug. The bug has been fixed, which is why it's in the bugfixes section, so there's no need to mention that in this description.

Reply:
changed